### PR TITLE
Aiko branch

### PR DESCRIPTION
**Pull request by aikotakata**
updated matches page and added some formatting

### CODE REVIEW

File: src/components/Matches.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { connect } from 'react-redux';
+import { } from '../actions';
+import './Matches.scss';
+import plus from '../assets/matches/add.png';
+import dog from '../assets/feed/dog.jpg';
+
+
+class Matches extends Component {
+  componentDidMount() {
+    console.log('loading');
```
Comment (hpfoster9) [inline src/components/Matches.js:14]: Moving forward we should probably try to limit the number of console logs we keep in PRs

File: src/components/Matches.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <div className="profile-box">Varsha Iyer</div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+const mapStateToProps = state => (
+  {
+    surveyAnswers: state.survey.answers,
```
Comment (hpfoster9) [inline src/components/Matches.js:58]: Since we aren't using any of these at the moment we probably shouldn't load them

File: src/environment.js [hunk 3]
```diff
@@ -2,9 +2,9 @@ const prod = {
   url: 'https://dartmate-backend.herokuapp.com',
 };
 
-const dev = {
-  url: 'http://localhost:3000',
-};
+// const dev = {
```
Comment (hpfoster9) [inline src/environment.js:5]: We shouldn't be adding this bit. Could you switch this back to normal and re-push?

**Review by hpfoster9 — CHANGES_REQUESTED** [review]

**Review by hpfoster9 — APPROVED** [review]

**Comment by aikotakata** [comment]
creating new pull request with fixes

**Pull request merged**